example/test_problem/Hydro/CDM_LSS/Input__DumpTable

```diff
@@ -0,0 +1,22 @@
+#Dump ID             Dump Time
+       0    0.00990099009900990099
+       1    0.0208596384137056
+       2    0.0476190476190476
+       3    0.0625000000000000
+       4    0.0666666666666667
+       5    0.0714285714285714
+       6    0.0769230769230769
+       7    0.0833333333333333
```

Comment:
I would recommend combining these 8 dumps with the original dump table `example/input/Input__DumpTable`. It will give us more dumps and a nicer movie.